docs/github.md

```diff
@@ -0,0 +1,41 @@
+# Using Jujutsu with Github Projects
+
+## Setup a ssh key
+As of December 2022 it's necessary to setup a ssh key to work with github
+projects. See [Github's Tutorial][gh]. This restriction may be lifted in the 
+future, see issue #469 for more information and progress on authenticated http. 
+
+
+## Basic workflows
+Jujutsu supports both a `git` and a `hg` like workflows. This allows you
+to continue using your favorite way of using the vcs. It's also possible to 
+just commit multiple changes and set the expected branch retroactively. 
+
+### Git workflow
+
+```shell script
+$ jj branch c $your-feature
+$ # Do your work
+$ jj commit -m "feat(xy): some thing"
```

Comment:
We should add a note somewhere explaining that the branch doesn't move forward if the user were to create more commits on top.

Reply:
Added.